astroplan/core.py

```diff
@@ -161,6 +163,46 @@ def altaz(self, time, target=None, obswl=None):
                 coordinate = target
             return coordinate.transform_to(altaz_frame)
 
+    def parallactic_angle(self, time, target):
+        '''
+        Calculate the parallactic angle.
+
+        Parameters
+        ----------
+        time : `~astropy.time.Time`
+            The time at which the observation is taking place.
+
+        target : {`~astroplan.FixedTarget`, `~astropy.coordinates.SkyCoord`}
```

Comment:
The braces `{` are for multiple-choice lists, usually strings like `{"fast", "slow", "turtle"}`
If multiple input types are accepted, remove the braches and add an or between the types (I think that's the common format ... check a few Astropy docstrings or the Numpy docstring guide.)